lua_remove undeclared on Lua 5.3.4

Hi,

I tried to install this from luarocks and i get the following error:
```
Installing http://luarocks.org/dev/luaffi-scm-1.src.rock                              
gcc -O2 -fPIC -I/usr/include -c call.c -o call.o -Idynasm                             
call.c: In function ‘reserve_code’:        
call.c:184:31: error: ‘lua_remove’ undeclared (first use in this function); did you mean ‘lua_xmove’?
         ADDFUNC(jit->lua_dll, lua_remove);                                           
                               ^
call.c:149:43: note: in definition of macro ‘ADDFUNC’                                 
         func = func ? func : (cfunction) &NAME; \                                    
                                           ^~~~
call.c:184:31: note: each undeclared identifier is reported only once for each function it appears in
         ADDFUNC(jit->lua_dll, lua_remove);                                           
                               ^
call.c:149:43: note: in definition of macro ‘ADDFUNC’                                 
         func = func ? func : (cfunction) &NAME; \                                    
                                           ^~~~

Error: Build error: Failed compiling object call.o
```

Reading through the issue history, it it should just be chaning [ffi.h#L108](https://github.com/facebook/luaffifb/blob/master/ffi.h#L108) to support newer versions.

Im using PUC Lua 5.3.4

--- a/ffi.h
+++ b/ffi.h
@@ -105,7 +105,7 @@ static char* luaL_prepbuffsize(luaL_Buffer* B, size_t sz) {
     }
     return luaL_prepbuffer(B);
 }
-#elif LUA_VERSION_NUM == 503
+#elif LUA_VERSION_NUM >= 503
 static void (lua_remove)(lua_State *L, int idx) {
     lua_remove(L, idx);
 }

